### PR TITLE
Add guide for tests specs

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

File: source/using/test-specs.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: Test Specifications with CocoaPods
+description: This is a guide for working with test specifications and CocoaPods
+order: 5
```
Comment (segiddins) [inline source/using/test-specs.md]: I think you need a `---` after this

Comment (dnkoutso) [inline source/using/test-specs.md]: you are right! fixed.

**Comment by dnkoutso** [comment]
Slightly updated to include `:testspecs => ['Tests']` support.

**Comment by orta** [comment]
Added all those ^ words: https://github.com/CocoaPods/peril-settings/commit/aad763db82bd3a3689b27365b7386784bb490583

**Pull request merged**